samples/word/01-basics/basic-api-call.yaml

```diff
@@ -1,7 +1,6 @@
-order: 1
```

Comment:
i already moved a bunch of the samples to other folders. The samples that are on basics, are really the basics. I will adjust the order of some of them.